examples/client/getBlockChainParameters.ts

```diff
@@ -0,0 +1,63 @@
+import { ChainParameters } from '@concordium/common-sdk';
+import { createConcordiumClient } from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn ts-node <path-to-this-file> [options]
+
+  Required
+    --account, -a  The account to get transactions from
+
+  Options
+    --help,     -h  Displays this message
+    --endpoint, -e  Specify endpoint of the form "address:port", defaults to localhost:20000
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            endpoint: {
+                type: 'string',
+                alias: 'e',
+                default: 'localhost:20000',
+            },
+            account: {
+                type: 'string',
+                alias: 'a',
+                isRequired: true,
+            },
+            blockhash: {
+                type: 'string',
+                alias: 'b',
+                default: '', // This defaults to LastFinal
+            },
+        },
+    }
+);
+
+const [address, port] = cli.flags.endpoint.split(':');
+const client = createConcordiumClient(
+    address,
+    Number(port),
+    credentials.createInsecure(),
+    { timeout: 15000 }
+);
+
+if (cli.flags.h) {
+    cli.showHelp();
+}
```

Comment:
This is a minor thing/I should have noticed before, shouldn't it exit here? (So that we only get the help output when asking for help) :shrug:

Reply:
Yes, I'll do a `sed` and fix it in #164

Reply:
Nevermind it works as intended